#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_debpaths.dpatch by  <richardaburton@hotmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.


@DPATCH@

diff -ur kicad-0.0.20060626.orig/common/gestfich.cpp kicad-0.0.20060626/common/gestfich.cpp
--- kicad-0.0.20060626.orig/common/gestfich.cpp	2006-01-31 16:25:48.000000000 +0000
+++ kicad-0.0.20060626/common/gestfich.cpp	2006-06-28 13:12:26.000000000 +0100
@@ -35,7 +35,7 @@
 	wxT("c:/kicad/help/"),
 	wxT("d:/kicad/help/"),
 #else
-	wxT("/usr/local/kicad/help/"),
+	//wxT("/usr/local/kicad/help/"),
 	wxT("/usr/share/doc/kicad/"),
 #endif
 	wxT("end_list")	// End of list symbol, do not change
@@ -47,7 +47,7 @@
 	wxT("c:/kicad/"),
 	wxT("d:/kicad/"),
 #else
-	wxT("/usr/local/kicad/"),
+	//wxT("/usr/local/kicad/"),
 	wxT("/usr/share/kicad/"),
 #endif
 	wxT("end_list")	// End of list symbol, do not change
@@ -59,8 +59,8 @@
 	wxT("c:/kicad/winexe/"),
 	wxT("d:/kicad/winexe/"),
 #else
-	wxT("/usr/local/kicad/linux/"),
-	wxT("/usr/local/bin/"),
+	//wxT("/usr/local/kicad/linux/"),
+	//wxT("/usr/local/bin/"),
 	wxT("/usr/bin/"),
 #endif
 	wxT("end_list")	// End of list symbol, do not change
@@ -402,7 +402,7 @@
 	int ii = 0;
 	while ( 1 )
 	{
-		if ( s_KicadDataPathList[ii] == wxT("end_list") ) break;
+		if ( s_KicadBinaryPathList[ii] == wxT("end_list") ) break;
 		FullFileName = s_KicadBinaryPathList[ii++] + shortname;
 		if ( wxFileExists(FullFileName) ) return FullFileName;
 	}
